tests/aktualizr_secondary/uptane_test.cc

```diff
@@ -1,6 +1,9 @@
 #include <gtest/gtest.h>
 
 #include "aktualizr_secondary/aktualizr_secondary.h"
+#include "sotauptaneclient.h"
+
+#include "../httpfake.h"
```

Comment:
Would it be better to tell CMake to search for header files in the tests directory (instead of using a relative path)?

Reply:
Yes thanks, I just added the directory to the include list of all the tests